dashboard/app/api.go

```diff
@@ -678,18 +678,32 @@ func apiReportBuildError(c context.Context, ns string, r *http.Request, payload
 	if err != nil {
 		return nil, err
 	}
-	if err := updateManager(c, ns, req.Build.Manager, func(mgr *Manager, stats *ManagerStats) error {
+	mgr, err := updateManager(c, ns, req.Build.Manager, func(mgr *Manager, stats *ManagerStats) error {
 		log.Infof(c, "failed build on %v: kernel=%v", req.Build.Manager, req.Build.KernelCommit)
 		if req.Build.KernelCommit != "" {
 			mgr.FailedBuildBug = bug.keyHash(c)
 		} else {
 			mgr.FailedSyzBuildBug = bug.keyHash(c)
 		}
 		return nil
-	}); err != nil {
+	})
+	if err != nil {
 		return nil, err
 	}
-	return nil, nil
+	// Determine whether we should continue fuzzing on this syz-manager.
+	continueFuzzing := false
+	if mgr.CurrentBuild != "" {
```

Comment:
Why can the mgr.CurrentBuild be ""?

Reply:
It can be for new managers for which we have no successful builds yet.